vars/piperInitRunStageConfiguration.groovy

```diff
@@ -57,60 +56,38 @@ void call(Map parameters = [:]) {
         .withMandatoryProperty('stageConfigResource')
         .use()
 
+    // Go logic to check if the step is active
+    String piperGoPath = parameters?.piperGoPath ?: './piper'
+    def resource = libraryResource(config.stageConfigResource)
+    config.stages = (readYaml(text: resource)).spec.stages
+    writeFile(file: ".pipeline/stage_conditions.yaml", text: resource)
+    def success = piperExecuteBin.checkIfStepActive(script,piperGoPath,".pipeline/stage_conditions.yaml",".pipeline/step_out.json",".pipeline/stage_out.json","_","_")
```

Comment:
Does not match the method signature `checkIfStepActive(Map parameters = [:],Script script,..`.

Reply:
```suggestion
    def success = piperExecuteBin.checkIfStepActive([:], script, piperGoPath, ".pipeline/stage_conditions.yaml", ".pipeline/step_out.json", ".pipeline/stage_out.json", "_", "_")
```

Reply:
Fixed